### PR TITLE
Add engineering-team-principles of List

### PR DESCRIPTION
**Pull request by posquit0**
**https://github.com/posquit0/awesome-engineering-team-principles**

**A curated list of awesome resources for engineering team principles**

### By submitting this pull request I confirm I've read and complied with the below requirements 🖖

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

## Requirements for your pull request

- I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- This pull request has a descriptive title.<br>For example, `Add Name of List`, not `Update readme.md` or `Add awesome list`.
- The entry in the Awesome list should:
	- Include a short description about the project/theme of the list. **It should not describe the list itself.**<br>Example: `- [Fish](…) - User-friendly shell.`, not `- [Fish](…) - Resources for Fish.`.
	- Be added at the bottom of the appropriate category.
- The list I'm submitting complies with these requirements:

## Requirements for your Awesome list

- **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
- It's the result of hard work and the best I could possibly produce.
- Non-generated Markdown file in a GitHub repo.
- **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
- The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
- Not a duplicate.
- Only has awesome items. Awesome lists are curations of the best, not everything.
- Includes a project logo/illustration whenever possible.
	- Either fullwidth or placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
	- The image should link to the project website or any relevant website.
	- The image should be high-DPI. Set it to maximum half the width of the original image.
- Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
- Includes the [Awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge).
	- Should be placed on the right side of the readme heading.
	- Should link back to this list.
- Has a Table of Contents section.
	- Should be named `Contents`, not `Table of Contents`.
	- Should be the first section in the list.
	- Should only have one level of sub-lists, preferably none.
- Has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license).
	- That means something like CC0, **not a code licence like MIT, BSD, Apache, etc.**
	- [WTFPL](http://www.wtfpl.net) and [Unlicense](http://unlicense.org) are not acceptable licenses.
	- If you use a license badge, it should be SVG, not PNG.
- Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines).
	- The file should be named `contributing.md`. Casing is up to you.
- Has consistent formatting and proper spelling/grammar.
	- The link and description are separated by a dash. <br>Example: `- [AVA](…) - JavaScript test runner.`
	- The description starts with an uppercase character and ends with a period.
	- Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
- Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.

Go to the top and read it again.

### CODE REVIEW

**Comment by posquit0** [comment]
As you know, awesome list is a curated list. I think there are a few **high-quality contents** about the engineering team principles on the Internet. :)

**Comment by sindresorhus** [comment]
```
❯ awesome-lint https://github.com/posquit0/awesome-engineering-team-principles
✖ Linting

  README.md:49:1
  ✖   30:1    ToC missing item for "Self Promotion"                                                      remark-lint:awesome/toc
  ✖   31:3    ToC missing item for "Google AI"                                                           remark-lint:awesome/toc
  ✖   42:3    ToC item "Discussions" link "#discussion" not found                                        remark-lint:awesome/toc
  ✖   49:1    Marker style should be *                                                                   remark-lint:unordered-list-marker-style
  ✖   49:72   List item description must end with proper punctuation                                     remark-lint:awesome/list-item
  ✖   53:1    Marker style should be *                                                                   remark-lint:unordered-list-marker-style
  ✖   53:74   List item description must end with proper punctuation                                     remark-lint:awesome/list-item
  ✖   57:1    Marker style should be *                                                                   remark-lint:unordered-list-marker-style
  ✖   57:3    Link to https://blog.imgur.com/2018/02/13/guiding-principles-for-imgur-engineers/ is dead  remark-lint:no-dead-urls
  ✖   57:87   List item description must end with proper punctuation                                     remark-lint:awesome/list-item
  ✖   61:1    Marker style should be *                                                                   remark-lint:unordered-list-marker-style
  ✖   61:81   List item description must end with proper punctuation                                     remark-lint:awesome/list-item
  ✖   62:1    Marker style should be *                                                                   remark-lint:unordered-list-marker-style
  ✖   66:1    Marker style should be *                                                                   remark-lint:unordered-list-marker-style
  ✖   66:82   List item description must end with proper punctuation                                     remark-lint:awesome/list-item
  ✖   67:1    Marker style should be *                                                                   remark-lint:unordered-list-marker-style
  ✖   68:1    Marker style should be *                                                                   remark-lint:unordered-list-marker-style
  ✖   72:1    Marker style should be *                                                                   remark-lint:unordered-list-marker-style
  ✖   76:1    Marker style should be *                                                                   remark-lint:unordered-list-marker-style
  ✖   76:103  List item description must end with proper punctuation                                     remark-lint:awesome/list-item
  ✖   80:1    Marker style should be *                                                                   remark-lint:unordered-list-marker-style
  ✖   80:58   List item description must end with proper punctuation                                     remark-lint:awesome/list-item
  ✖   87:1    Marker style should be *                                                                   remark-lint:unordered-list-marker-style
  ✖   87:139  List item description must end with proper punctuation                                     remark-lint:awesome/list-item
  ✖   88:1    Marker style should be *                                                                   remark-lint:unordered-list-marker-style
  ✖   88:152  List item description must end with proper punctuation                                     remark-lint:awesome/list-item
  ✖   89:1    Marker style should be *                                                                   remark-lint:unordered-list-marker-style
  ✖   89:162  List item description must end with proper punctuation                                     remark-lint:awesome/list-item
  ✖   90:1    Marker style should be *                                                                   remark-lint:unordered-list-marker-style
  ✖   90:141  List item description must end with proper punctuation                                     remark-lint:awesome/list-item
  ✖   94:1    Marker style should be *                                                                   remark-lint:unordered-list-marker-style
  ✖   94:161  List item link and description must be separated with a dash                               remark-lint:awesome/list-item
  ✖   98:1    Marker style should be *                                                                   remark-lint:unordered-list-marker-style
  ✖  102:1    Marker style should be *                                                                   remark-lint:unordered-list-marker-style

  34 errors
```

https://github.com/sindresorhus/awesome-lint

**Comment by sindresorhus** [comment]
Can you fix the merge conflict?

**Comment by sindresorhus** [comment]
> I think there are a few **high-quality contents** about the engineering team principles on the Internet.

I think you haven't looked hard enough. I refuse to believe there's no a single video of it on the internet: https://github.com/posquit0/awesome-engineering-team-principles#videos

File: readme.md [hunk 1]
```diff
@@ -468,6 +468,7 @@
 - [Productivity](https://github.com/jyguyomarch/awesome-productivity#readme)
 - [Niche Job Boards](https://github.com/tramcar/awesome-job-boards#readme)
 - [Programming Interviews](https://github.com/MaximAbramchuck/awesome-interview-questions#readme)
+- [Engineering Team Principles](https://github.com/posquit0/awesome-engineering-team-principles#readme)
```
Comment (sindresorhus) [inline readme.md]: > The entry in the Awesome list should:
* Include a short description about the project/theme of the list. **It should not describe the list itself.**

Comment (posquit0) [reply readme.md]: I resolved the conflict and added the description.

**Comment by posquit0** [comment]
I fixed almost lint errors with `awesome-lint`. However, I think some lint errors can be ignored.
And `awesome-lint` says like the below. But the link is live.

```
  ✖   59:3    Link to https://blog.imgur.com/2018/02/13/guiding-principles-for-imgur-engineers/ is dead  remark-lint:no-dead-urls
```

**Comment by sindresorhus** [comment]
> I think you haven't looked hard enough. I refuse to believe there's no a single video of it on the internet: posquit0/awesome-engineering-team-principles#videos

I think this list still needs more content. The Table of Contents is almost larger than the actual content.

**Comment by sindresorhus** [comment]
> I think this list still needs more content.

Closing for now for the above reason. Try again in the future when your Awesome list is more mature and comprehensive.

**Pull request closed without merging**